packages/discovery-provider/src/challenges/listen_streak_endless_challenge.py

```diff
@@ -91,7 +91,7 @@ def generate_specifier(self, session: Session, user_id: int, extra: Dict) -> str
         # Otherwise, create a new specifier
         created_at = datetime.fromtimestamp(extra["created_at"])
         formatted_date = created_at.strftime("%Y%m%d")
-        if env == "stage":
+        if env == "stage" or env == "dev":
```

Comment:
don't think any of these changes are intended to be commited?

Reply:
Let's check with Reed on this one. @dharit-tan

Reply:
yes sry dont commit the `=="dev"` changes in this file plz! my b dylan

Reply:
yea can just revert all of the changes in this file